src/lib/sort/sort-header.ts

```diff
@@ -113,4 +114,16 @@ export class MdSortHeader implements MdSortable {
   _isSorted() {
     return this._sort.active == this.id && this._sort.direction;
   }
+
+  /**
+   * Returns the aria-sort attribute that should be applied to this sort header. If this header
+   * is not sorted, returns null so that the attribute is removed from the host element. Aria spec
+   * says that the aria-sort property should only be present on one header at a time, so removing
+   * ensures this is true.
+   */
+  _getAriaSortAttribute() {
+    if (!this._isSorted()) { return null; }
```

Comment:
Shouldn't it be `none` when it's not sorted?

Reply:
According to the spec, `For each table or grid, authors SHOULD apply aria-sort to only one header at a time.` so by returning null, the attribute is removed altogether rather than being set to `'none'` since it's possible another header now has `aria-sort` applied